### PR TITLE
Add road runner to the dictionary

### PR DESCRIPTION
**Pull request by ibds-student**
Is this OK?

### CODE REVIEW

File: people.py [hunk 1]
```diff
@@ -22,6 +22,14 @@ def _add_oliver(people):
     key = new_p.first_name + '_' + new_p.surname
     people[key] = new_p
 
+
+def _add_road_runner(people):
+    """ adds oliver """
```
Comment (osmart) [inline people.py]: This docstring looks wrong surely
```
""" adds Road Runner """
```

**Review by osmart — CHANGES_REQUESTED** [review]

File: people.py [hunk 2]
```diff
@@ -22,6 +22,14 @@ def _add_oliver(people):
...
+    new_p = PERSON(first_name='Road', surname='Runner',
+                   age=71, pet=None, hobbies=['running', 'seed'])
+    key = new_p.first_name + '_' + new_p.surname
+    people[key] = new_p
+
```
Comment (osmart) [inline people.py:32]: PEP8 says there should be two blank lines here

Comment (ibds-student) [reply people.py:32]: fixed in 9d6c0ba 

@osmart is this OK?

File: people.py (hunk 1 above)
Comment (ibds-student) [reply people.py]: Fixed in 691dc90

**Review by osmart — APPROVED** [review]

**Comment by osmart** [comment]
Merge road runner from ibds-student

**Pull request merged**